var/spack/repos/builtin/packages/boost/fee1ad07968386b6d547f089311b7a2c1bf7fa55.patch

```diff
@@ -0,0 +1,28 @@
+From fee1ad07968386b6d547f089311b7a2c1bf7fa55 Mon Sep 17 00:00:00 2001
```

Comment:
Can we remove this file and use a remote patch instead?

Reply:
Removed thanks to this: https://github.com/spack/spack/pull/25534#discussion_r698363950.